### ISSUE TITLE
Clarify CCDD API import in scripts

### ISSUE THREAD

**Issue by ascended121** [nasa/CCDD#28]
All of the example python scripts shipped with CCDD include the following line at the top of the file:
```
from CCDD import CcddScriptDataAccessHandler
```
however, the module imported (CcddScriptDataAccessHandler) doesn't appear to be used in any of the scripts. Further, the `ccdd` variable appears to be defined by default no matter if the import is used or not. The script shown here:
```
with open("init.txt","w") as file:
    file.write(",".join(dir()))
    file.write("\n")
    file.write(str(ccdd))
    file.write("\n")
    file.write(ccdd.getScriptName())
```

produces the following output:
```
__builtins__,ccdd,ccdds,file
CCDD.CcddScriptDataAccessHandler@3c874e39
C:\myCCDDpath\CCDD\scripts\testImport.py
```
and demonstrates that the variables ccdd and ccdds are created without the need for the import.

Could you clarify the purpose of the CcddScriptDataAccessHandler import? 

Additionally, could you provide details on how to import the ccdd variable into other user-provided modules? Some of my scripts have reached a size where it would be preferable to organize them in multiple files, but I haven't been able to figure out what I need to import in the support files to have access to the same ccdd variable that the main script gets automatically. 

It would be much preferable to be able to use something like 
```
from CCDD.CcddScriptHandler import accessHandler as ccdd
```
and be able to use it in any module called by the script than to have the ccdd variable defined implicitly only in the main script. From my (limited) knowledge of jython, the above statement would work if accessHandler was a public variable in the CcddScriptHandler class and would provide the same ccdd variable as currently defined.

**Comment by KevinMcCluney** [comment]
The import is included so that the script data access methods are available to a script.  Python is unique (at least for the script languages I've tested) in that the import statement isn't necessary.  I can't be certain that is true for all versions of Python/Jython, Java, etc.

You can access the script data access methods from a script other than the main one called from CCDD.  I could not get the method you suggested to work.  Here's how I did it:
1) Create a folder representing the python script package.
2) Place the python scripts in the package folder.  The main script doesn't have to be in the package folder, but the child scripts do.
3) Add the file __init__.py in the package folder.  In this file add import statements in the form 'from <i>child</i> import *' where <i>child</i> is the child script (minus the .py extension).
4) In the child scripts add 'from CCDD import CcddScriptDataAccessHandlerStatic as ccdd'.  Notice that this must be a reference to the static version of the access method class!  No import class is needed in the main script (as noted above).
5) In the main script add 'import sys', 'sys.path.append("<i>path to the package folder</i>")', then 'import <i>package folder name</i>'.  The system path must include the package folder's location and the update must occur before importing the package.
6) To call a function in a child script from the main script use the format '<i>package folder name.child function name</i>()'.

**Comment by ascended121** [comment]
Appreciate the info Kevin, I had tried importing CcddScriptDataAccessHandler (note the lack of static) and didn't have much luck... didn't realize the static was the one I wanted since it looks like the main script is provided an instance of CcddScriptDataAccessHandler. In any case, I now have it working and you've answered the questions, so I think this can be closed. Thanks!

As a note for others, jython comes with an installation of pip, and I was able to use the `-e` option to do a developer install of my package, allowing me to continue developing it but access it like it was an installed package, thus removing the need to modify the python path as noted in step 5 above.
